components/Footer.js

```diff
@@ -0,0 +1,56 @@
+import styled from "styled-components";
+import { useRouter } from "next/router";
+import { BiHomeHeart } from "react-icons/bi";
+import { RiAddLine } from "react-icons/ri";
+import { BsJournalText } from "react-icons/bs";
+
+export default function Footer() {
+  const router = useRouter();
+
+  return (
+    <FooterWrapper>
+      <IconWrapper onClick={() => router.push("/gamelist")}>
```

Comment:
You should specify the type attribute for the `<button>` element. _Different browsers may use different default types for the `<button>` element._  … w3scools.com says.